rstudio/rstudio
# Improve version clarity in About dialog

### Intent

This change is intended to improve the clarity of the version information in the About dialog. 

Before:

![image](https://user-images.githubusercontent.com/470418/128229684-88c3ec37-f659-495f-8c2e-f39e4fe2616e.png)

After:

![image](https://user-images.githubusercontent.com/470418/128229579-a477d6d8-7730-4876-94a1-bcf6bb81e096.png)

Addresses https://github.com/rstudio/rstudio/issues/9675. 
Addresses https://github.com/rstudio/rstudio/issues/9677. 

### Approach

- Tweak font sizes, weights, colors, and padding so that the most important part of the version, 2021-07.0 in this example, stands out the most.
- Add a "Pre-Release Build" banner to non-release builds to make it instantly obvious that they are not stable releases.
- Add a button to copy version information to the clipboard. 

### Automated Tests

N/A

### QA Notes

This change is pretty much style-only; the only new functionality here is the Copy Version button, which should ... copy the version. 

### Checklist
